jetty-server/src/main/java/org/eclipse/jetty/server/HttpConfiguration.java

```diff
@@ -318,11 +318,21 @@ public boolean getSendXPoweredBy()
         return _sendXPoweredBy;
     }
 
+    /**
+     * Indicates if the {@code Date} header should be sent in responses.
```

Comment:
This is the HTTP specified standard header `Date`.
From https://tools.ietf.org/html/rfc7231#section-7.1.1.2
Registered at https://www.iana.org/assignments/message-headers/message-headers.xhtml

Too many people will read this javadoc and assume that `Date` is optional, superfluous, and something that Jetty just decided to do.   

Please add the following ...

``` java
@see <a href="https://tools.ietf.org/html/rfc7231#section-7.1.1.2">HTTP/1.1 Standard Header: Date</a>
```

Reply:
Updated. Please review.

Reply:
Cool. much improved.
Thanks.